### ISSUE TITLE
Idea for exposing the raw video stream on a GPIO pin.

### ISSUE THREAD

**Issue by SziluChannel** [jeanlemotan/esp32-cam-fpv#35]
I would like to use the esp32 for encoding the video but use a separate VHF band RF module for sending and receiving the data. In this case turning off the wifi transmission would be a good option power wise.
Of course this communication would be only monodirectional thus setting some default values in code for the resolution and values like that would be neccessary.

**Comment by jeanlemotan** [comment]
There are several issues here: 
1. Video is jpeg compressed. You can ask the camera for uncompressed video, but the FPS would be horrible due to bandwidth issues
2. GPIO is digital. The ESP32 has a DAC for analog output, but and there are ppl that managed to output composite video from the DAC AFAIK, but it requires a lot of CPU to do that. And I'm not sure the DAC pins are not already used by the camera interface.

Basically what you want has not much to do with the current project. You want to take the digital, uncompressed output of the camera and make it analog composite. Very different than the goals of this project.
